docs/installation/cloud-ex-aws.md

```diff
@@ -0,0 +1,196 @@
+[menu.main]
+parent = "smn_cloud"
++++
+<![end-metadata]-->
+
+# Example: Manual install on a cloud provider
+
+You can install Docker Engine directly to servers you have on cloud providers.  This example shows how to create an <a href="https://aws.amazon.com/" target="_blank"> Amazon Web Services (AWS)</a> EC2 instance, and install Docker Engine on it.
+
+You can use this same general approach to create Dockerized hosts on other cloud providers.
+
+### Step 1. Sign up for AWS
+
+1. If you are not already an AWS user, sign up for <a href="https://aws.amazon.com/" target="_blank"> AWS</a> to create an account and get root access to EC2 cloud computers. If you have an Amazon account, you can use it as your root user account.
+
+2. Create an IAM (Identity and Access Management) administrator user, an admin group, and a key pair associated with a region (if you have not done so). (From the AWS menus, select to **Services &rarr; IAM**).
+
+  See the AWS documentation on <a href="http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/get-set-up-for-amazon-ec2.html" target="_blank">Setting Up with Amazon EC2</a>. Follow the steps for "Create an IAM User" and "Create a Key Pair". You will use these credentials for AWS manage VMs and
+
+  If you are just getting started with AWS and EC2, you do not need to create a virtual private cloud (VPC) or specify a subnet. The newer EC2-VPC platform (accounts created after 2013-12-04) comes with a default VPC and subnet in each availability zone. When you launch an instance, it automatically uses the default VPC.
+
+### Step 2. Configure and start an EC2 instance
+
+Launch an instance to create a virtual machine (VM) with a specified operating system (OS) as follows.
+
+  1. Log into AWS with your IAM credentials.
+
+      On the AWS home page, click **EC2** to go to the dashboard, then click **Launch Instance**.
+
+      ![EC2 dashboard](images/ec2_launch_instance.png)
+
+      AWS EC2 virtual servers are called *instances* in Amazon parlance. Once you set up an account, IAM user and key pair, you are ready to launch an instance. It is at this point that you select the OS for the VM.
+
+  2. Choose an Amazon Machine Image (AMI) with the OS and applications you want. For this example, we select an Ubuntu server.
+
+      ![Launch Ubuntu](images/ec2-ubuntu.png)
+
+  3. Choose an instance type.
+
+      ![Choose a general purpose instance type](images/ec2_instance_type.png)
+
+  * Configure the instance.
+
+    You can select the default network and subnet, which are inherently linked to a region and availability zone.
+
+      ![Configure the instance](images/ec2_instance_details.png)
+
+  * Click **Review and Launch**.
+
+  * Select a key pair to use for this instance. (When you choose to launch, you need to select a key pair to use. Save the `.pem` file to use in the next steps. Save the )
+
+The instance is now up-and-running. The link path to get back to your EC2 instance on AWS is:
```

Comment:
"link path" maybe menu path?  I'm not sure putting the path in the code font is working well. It looks like something I cut and paste but it isn't.